Makefile

```diff
@@ -24,6 +24,9 @@ DOCKER_COMPOSE_BIN := $(shell command -v $(DOCKER_COMPOSE_BIN_NAME) 2> /dev/null
 DOCKER_BIN := $(shell command -v $(DOCKER_BIN_NAME) 2> /dev/null)
 MINIMOCK_BIN=$(VENDOR_DIR)/github.com/gojuno/minimock/cmd/minimock/minimock
 
+# Define and get the vakue for UNAME_S variable from shell
+UNAME_S := $(shell uname -s)
```

Comment:
What is this used for @rohitkrai03 ?

Reply:
@kwk This variable is used to check for the kernel of operating system and then decide which version of dep package to install. On macOS when i tried `make build` it failed because there was no `UNAME_S` variable set which is why a version of `dep` targeted for linux was getting downloaded. I could only run `make build` succesfully after running`export UNAME=Darwin`

Reply:
Ah, I see. Thank you. I was wondering about this because it wasn't explicitly used.